Util/CellUtils/squashCells.m

```diff
@@ -0,0 +1,86 @@
+function [mcell] = squashCells(varargin)
```

Comment:
i'm lost of the purpose of this function but sure

Reply:
This was created because I had to create a varargin cell with alternate inputs from two cells with arbritrary sizes. this function is a poor man kwargs, but for matlab.

I faced a access pattern that I had to build a {"key0","value0","key1","value1",...} cell to pass as varargin to other matlab functions (see ```discover_data_dimensions```). I actually faced this before but this time I couldn't run away from it because items were dynamic.

This also allows this kind of code:
```
f(arg1,arg2,"myoption1",'A',"myoption2",True);
```

to be written as:
```
opt_names = {'myoption1","myopyion2"};
opt_values = {'A',True};
extra_opts = squashCells(opt_names,opt_values);
f(arg1,arg2,extra_opts{:});
```

Although this appears inferior (4 lines instead of 1), the above is one only way when the options are dynamically generated within the current scope. Moreover, the above is way more readable when you have a lot of options or when changes came through (e.g. diffs). Finally, when you are debugging it is way more easier to manipulate opt_names/values to test the function call than to modify by hand the overly long function call.